test/mock.test.js

```diff
@@ -45,7 +45,7 @@ describe('stubbing', () => {
 
     AWS.clearAllMocks();
     const after = new AWS.S3();
-    expect(jest.isMockFunction(after.putObject)).toEqual(false);
+    expect(jest.isMockFunction('after.putObject')).toEqual(false);
```

Comment:
What is this change for?

Reply:
Oh, this needs to be removed. I was tinkering to see what `isMockFunction` did for values that aren't function (answer: it returns `false`).